astroquery/esa/xmm_newton/tests/test_xmm_newton.py

```diff
@@ -64,6 +68,15 @@ def test_get_postcard(self):
         xsa = XMMNewtonClass(self.get_dummy_tap_handler())
         xsa.get_postcard(**parameters)
 
+    @pytest.mark.remote_data
```

Comment:
This test file gets nicely long, maybe it's time to factor out the remote tests to a separate `test_xmm_newton_remote.py`. (See most other modules have a `test_modulename.py` and a `test_modulename_remote.py`, it makes it's easier to spot issues. Also, when it makes sense tests can be duplicated between the two, e.g. we always run the mocked local ones, but only cron/merge run the remote ones. The previous tests that the code internally consistent, but it doesn't provide any safe guarding against remote API changes).

Reply:
on this line also - this is marked as "remote data", but it uses "dummy_tap_handler" - doesn't that mean it is not doing a remote request?